brainiak/reprsimil/brsa.py

```diff
@@ -1226,7 +2090,98 @@ def _fit_diagV_GP(
                 break
 
         return current_vec_U_chlsk_l, current_a1, current_logSNR2,\
-            current_GP, X0
+            current_GP, X_res
+
+    def _fit_null(self, Y, X_base, scan_onsets=None):
```

Comment:
For future refactoring it may be possible to just call `fit` here with a design matrix of 1's rather than have separate method (though it will require changing the error checking in `fit` also).